webpack/scenes/Hosts/ChangeContentSource/actions.js

```diff
@@ -17,18 +17,20 @@ export const getFormData = (hostIds, search) => (post({
   errorToast: () => __('Something went wrong while getting the data. See the logs for more information'),
 }));
 
-export const changeContentSource = (environmentId, contentViewId, contentSourceId, hostIds) =>
-  put({
-    key: CHANGE_CONTENT_SOURCE,
-    url: foremanUrl('/api/v2/hosts/bulk/change_content_source'),
-    params: {
-      environment_id: environmentId,
-      content_view_id: contentViewId,
-      content_source_id: contentSourceId,
-      host_ids: hostIds,
-    },
-    errorToast: () => __('Something went wrong while updating the content source. See the logs for more information'),
-  });
+export const changeContentSource =
+  (environmentId, contentViewId, contentSourceId, hostIds, handleSuccess) =>
+    put({
+      key: CHANGE_CONTENT_SOURCE,
+      url: foremanUrl('/api/v2/hosts/bulk/change_content_source'),
+      params: {
+        environment_id: environmentId,
+        content_view_id: contentViewId,
+        content_source_id: contentSourceId,
+        host_ids: hostIds,
+      },
+      errorToast: () => __('Something went wrong while updating the content source. See the logs for more information'),
+      handleSuccess,
```

Comment:
Without this, were we not showing success toasts before?

Reply:
The handleSuccess is so it can redirect you to the job wizard. The flow is (1) click Run Job invocation; (2) it saves the content source/CV/LCE; (3) you are redirected to the job wizard so you can complete the changes on the hosts.